common/app/views/fragments/analytics/google.scala.html

```diff
@@ -82,7 +82,7 @@
                 var d = new Date().getTime();
                 if (d - referrerVars.value.time < 60 * 1000) { // One minute
                     ga('guardianTestPropertyTracker.send', 'event', 'Click', 'Internal', referrerVars.value.tag, {
-                        dimension12: referrerVars.value.pageName
+                        dimension12: referrerVars.value.path
```

Comment:
It's just GitHub getting confused by apostrophes. Feast your eyes on this! https://github.com/guardian/frontend/blob/b0b77d853601d280fcea29ea05b0f135e83245b3/common/app/views/fragments/analytics/google.scala.html